cpp/zotaut/dump_delivered_marc_records.cc

```diff
@@ -79,19 +80,29 @@ void GetJournalEntriesFromDb(DbConnection * const db_connection, const std::stri
             const auto dois(record.getDOIs());
             csv_row += ";" + TextUtil::CSVEscape(StringUtil::Join(dois, '\n'));
 
-            for (const auto &_936_field : record.getTagRange("936")) {
-                csv_row += ";" + _936_field.getFirstSubfieldWithCode('j');
-                csv_row += ";" + _936_field.getFirstSubfieldWithCode('d');
-                csv_row += ";" + _936_field.getFirstSubfieldWithCode('e');
-                break;
-            }
+            const auto issue_info(BSZUtil::ExtractYearVolumeIssue(record));
+
+            std::string data_year("");
+            if (not issue_info.year_.empty())
+                data_year = issue_info.year_;
+
+            std::string data_volume("");
+            if (not issue_info.volume_.empty())
+                data_volume = issue_info.volume_;
+
+            std::string data_issue("");
+            if (not issue_info.issue_.empty())
+                data_issue = issue_info.issue_;
+
+            csv_row += ";" + data_year;
```

Comment:
Now that's definitely better than before, however i suggest to also use TextUtil::CSVEscape on the variables just to be sure (in theory they might contain a semicolon in some cases).